core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/cluster/DatalakeResources.java

```diff
@@ -96,27 +104,54 @@ public void setDatalakeStackId(Long datalakeStackId) {
     }
 
     public String getDatalakeAmbariUrl() {
-        return datalakeAmbariUrl;
+        return isNotEmpty(getDatalakeClusterManagerUrl()) ? getDatalakeClusterManagerUrl() : datalakeAmbariUrl;
+    }
+
+    public String getDatalakeClusterManagerUrl() {
+        return datalakeClusterManagerUrl;
     }
 
     public void setDatalakeAmbariUrl(String datalakeAmbariUrl) {
         this.datalakeAmbariUrl = datalakeAmbariUrl;
+        setDatalakeClusterManagerUrl(datalakeAmbariUrl);
+    }
+
+    public void setDatalakeClusterManagerUrl(String datalakeClusterManagerUrl) {
+        this.datalakeClusterManagerUrl = datalakeClusterManagerUrl;
     }
 
     public String getDatalakeAmbariIp() {
-        return datalakeAmbariIp;
+        return isNotEmpty(getDatalakeClusterManagerIp()) ? getDatalakeClusterManagerIp() : datalakeAmbariIp;
+    }
+
+    public String getDatalakeClusterManagerIp() {
+        return datalakeClusterManagerIp;
     }
 
     public void setDatalakeAmbariIp(String datalakeAmbariIp) {
         this.datalakeAmbariIp = datalakeAmbariIp;
+        setDatalakeClusterManagerIp(datalakeAmbariIp);
+    }
+
+    public void setDatalakeClusterManagerIp(String datalakeClusterManagerIp) {
+        this.datalakeClusterManagerIp = datalakeClusterManagerIp;
     }
 
     public String getDatalakeAmbariFqdn() {
-        return datalakeAmbariFqdn;
+        return isNotEmpty(getDatalakeClusterManagerFqdn()) ? getDatalakeClusterManagerFqdn() : datalakeAmbariFqdn;
```

Comment:
Do we need DataLake...Fqdn? I have teh feeling that this might be removed completely

Reply:
Probably you're right, but I think removing/changing anything which is not - closely - related to the/this renaming task has no place here.
Would you like to have a ticket for it? If so, I'll create one and suggest that to put it into our future sprint.

Reply:
I mean, it does not make sense to rename if we are not using it at all. Then it can stay Ambari and in a next commit you can delete it.